app/imports/api/actionItems/server/publications.js

```diff
@@ -23,3 +23,19 @@ Meteor.publish('actionItems', function publishActionItems(sprintId) {
     }
     return this.ready();
 });
+
+Meteor.publish('projectActionItems', function publishProjectActionItems(projectId) {
```

Comment:
You can merge publications, eg `function publishActionItems(sprintOrProjectId, type = 'sprint') {`